### DISCUSSION TITLE
lidarr script doesn't load music videos

### DISCUSSION THREAD

**Discussion by BulldozerPete** [RandomNinjaAtk/arr-scripts#287] [Q&A] [answered]
Hello,

I have set up lidarr from linuxserver.io with the provided script here. I saw in the docker-log some downloads and installation running, when starting lidarr.

Now I have added some music and mapped that successfully. But I don't see any music video.

What am I doing wrong?

**Comment by BulldozerPete** [comment]
Is there any forum/ discord I can get on to get some support?

**Comment by jaycedk** [comment]
"Now I have added some music and mapped that successfully. But I don't see any music video."
@BulldozerPete Where do you expect to see the music videos ?
They should be downloaded to your, in config defined folder.

**Comment by BulldozerPete** [comment]
I have figured out, that I had to configure 
`videoPath=` and `addFeaturedVideoArtists="true"` too.

As well as `videoFormat="bv+ba/b"` otherwise no videos were downloaded.

A hint in the setup documentation would be helpful.

**Reply by agster27** [reply]
@BulldozerPete Can you give me an example of what you used in your extended.conf? I am also trying to get this to show up. What does the videoFormat="bv+ba/b" for?

**Comment by jaycedk** [comment, marked as answer]
All the hints are in the extended.conf
https://github.com/RandomNinjaAtk/arr-scripts/blob/main/lidarr/extended.conf
Did you read it ?

**Reply by agster27** [reply]
@jaycedk I am having a similar issue. I am happy to make edits on the Wiki once this is figured out. 

I have configured the extended.conf and its working great. Videos are downloading. However they are not showing up in plex. The piece I am missing is how to let plex know where the videos live. Any insight would be great.

**Reply by jaycedk** [reply]
I don't use Plex, so would not know.
My guess is map them in plex 🤷‍♂️

**Reply by agster27** [reply]
I figured it out. I wasn't sure if these scripts notified plex of the videos.

For any future search:
https://support.plex.tv/articles/205568377-adding-local-artist-and-music-videos/

**Comment by BulldozerPete** [comment]
I read this: https://github.com/RandomNinjaAtk/arr-scripts/blob/main/lidarr/readme.md

But why is the configuration optional, when it doesn't run out of the box?
My suggestion also would be to aggregate all music setting and video setting in the configuration. Would be more user-friendly.

**Reply by jaycedk** [reply]
I guess you are welcome to put in a pr with the corrections you would like to see.
Or add to the wiki.
https://github.com/RandomNinjaAtk/arr-scripts/wiki/Lidarr
